public/view/main.jsx

```diff
@@ -39,33 +41,38 @@ const SUPPORTED_COUNTRIES = ['br', 'co', 'pa'];
 // ugliness here because AppMain hasn't yet been instantiated.
 var main_instance = null;
 var map_controller = null;
-var rerender = function() {
+var _base_rerender = function() {
   if (main_instance !== null) {
     main_instance.forceUpdate();
   }
 };
-var rerender_and_redraw = function() {
-  rerender();
+var rerender = _.debounce(_base_rerender, 16);  // 16 milliseconds is 60 frames per second ... fast enough
+var rerender_and_redraw = _.debounce(function() {
+  _base_rerender();
   if (map_controller !== null) {
     map_controller.redraw();
   }
-};
+}, 16);
```

Comment:
Any idea how much debouncing saves us? That is, in practice, are we actually preventing redundant redraws?